pom.xml

```diff
@@ -183,14 +183,14 @@
     <!-- The surefire/failsafe test execution logs goes to a file by default -->
     <testLogToFile>true</testLogToFile>
     <version.com.github.spotbugs.spotbugs-maven-plugin>4.8.3.1</version.com.github.spotbugs.spotbugs-maven-plugin>
-    <version.compiler.plugin>3.8.0-jboss-2</version.compiler.plugin>
+    <version.compiler.plugin>3.13.0</version.compiler.plugin>
```

Comment:
Is it at all possible to just remove this line and use the version from the JBoss parent (https://github.com/jboss/jboss-parent-pom/blob/jboss-parent-39/pom.xml)?

Reply:
Yes, good point. I'll test it and update the PR if that works